<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Test Suite" duration-ms="8006" started-at="2017-09-14T11:39:48Z" finished-at="2017-09-14T11:39:56Z">
    <groups>
    </groups>
    <test name="FirstTestCase" duration-ms="8006" started-at="2017-09-14T11:39:48Z" finished-at="2017-09-14T11:39:56Z">
      <class name="com.phptravels.TestScripts.UserLogin">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.phptravels.TestScripts.UserLogin@146ba0ac]" name="setUp" is-config="true" duration-ms="10102" started-at="2017-09-14T17:09:38Z" finished-at="2017-09-14T17:09:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testLogin()[pri:0, instance:com.phptravels.TestScripts.UserLogin@146ba0ac]" name="testLogin" duration-ms="7999" started-at="2017-09-14T17:09:48Z" finished-at="2017-09-14T17:09:56Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.phptravels.locator.LocatorReader.getLocator(LocatorReader.java:30)
	at com.phptravels.pagehelper.LoginHelper.enterUserName(LoginHelper.java:32)
	at com.phptravels.TestScripts.UserLogin.testLogin(UserLogin.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method status="PASS" signature="afterMainMethod()[pri:0, instance:com.phptravels.TestScripts.UserLogin@146ba0ac]" name="afterMainMethod" is-config="true" duration-ms="0" started-at="2017-09-14T17:09:56Z" finished-at="2017-09-14T17:09:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMainMethod -->
      </class> <!-- com.phptravels.TestScripts.UserLogin -->
    </test> <!-- FirstTestCase -->
  </suite> <!-- Test Suite -->
</testng-results>
